.gitignore

```diff
@@ -9,3 +9,4 @@ _site/
 vendor/bundle/ruby
 .bundle/config.markdown
 vendor/bundle/ruby
+Gemfile.lock
```

Comment:
Is this supposed to be on gitignore? You have changes on this file in this PR and this file handles the versioning for the gems so I believe we don't want to start ignoring changes to it no?

Reply:
Yes in our case `Gemfile.lock` should be kept under version control to ease the development process.

Reply:
Read it from the Jekyll best practise doc to include it in gitignore.

Reply:
Since we deploy this app using Github pages it is not necessary to keep the `Gemfile.lock` under version control because it is [apparently not used](https://stackoverflow.com/a/56297531) by their process to build the site.

I believe @nashthecoder is referencing this source as well: https://ben.balter.com/jekyll-style-guide/gitignore/

So it comes down to a local development perspective if we keep it or not under version control. It is generally a good practice to have a lock file so multiple devs are not using different versions of dependencies to build apps.

Since the gems in the `Gemfile` are not used in production and for this project should almost only consist of jekyll and a webserver keeping the lock file out of version control should not have that much impact for local development.

(Somewhat related to this we should again revisit the local development process to review the webserver to use and to check if we are using the gems correctly. It seems that we should be using the `github-pages` gem to define the jekyll version to use on our project since we host via github pages. Source: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll#creating-your-site)